app/src/main/res/values/strings.xml

```diff
@@ -142,7 +142,7 @@
     <string name="nc_new_password">New password</string>
     <string name="nc_wrong_password">Wrong password</string>
     <string name="nc_about">About</string>
-    <string name="nc_privacy">Privacy</string>
+    <string name="nc_privacy">Privacy and Legal Policy</string>
```

Comment:
As far as is see there's no need to rename "Privacy" because the menu for master doesn't contain any other clickable entry like this? (like it would have been for v9)

i like the clarification of the items for the sound settings!

Reply:
You're right. I didn't notice it. I returned to the previous description 😉.

Reply:
Alternatively, I can change to a more professional name which is: `Privacy policy`.

Reply:
i think "Privacy" is fine. Files app also named it like this.